eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/Objectionary.java

```diff
@@ -40,4 +40,11 @@ public interface Objectionary {
      */
     Input get(String name) throws IOException;
 
+    /**
+     * Checks whether an Objectionary contains a provided object.
+     * @param name Object name.
+     * @return Object code.
```

Comment:
@MikhailLipanin It returns boolean. How is it connected with "code"?

Reply:
@volodya-lombrozo, It will be used in `ProbeMojo` to check the presence of an object [here](https://github.com/objectionary/eo/blob/408277c9eb53a33b59688b68a812ec8d1c3e41bd/eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java#L139-L143)

Reply:
fixed